kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/TlsSecrets.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.kubernetes.operator;
+
+import java.nio.file.Path;
+import java.util.List;
+
+import io.kroxylicious.kubernetes.api.v1alpha1.KafkaProxy;
+
+public record TlsSecrets(Path path, String volumeName, String secretName) {
+
+    private static final Path TLS_SECRETS_ROOT_DIR = Path.of("/opt/kroxylicious/secrets/tls");
+    private static final String CERTIFICATE = "tls.crt";
```

Comment:
maybe we should make these names configurable using the TLS options?

Reply:
you mean so we can work with secrets that expose different data keys?  If so, yes, but probably lower priority.